src/extensions/default/NavigationAndHistory/html/recentfiles-template.html

```diff
@@ -0,0 +1,165 @@
+<style>
```

Comment:
The styles should probably be in its own CSS file and then loaded with `ExtensionUtils.loadStyleSheet(module, "styles/recent-files.css");`